### ISSUE TITLE
[Bug]: Editing existing geometries not possible in 3.5.2

### ISSUE THREAD

**Issue by opbernardo** [3liz/lizmap-web-client#2884]
### What is the bug?

Hi,

I just installed Lizmap Webclient 3.5.2, and I found the edition tool It does not work properly...

With the tool, I can draw, lines, points and polygons, but I can not edit them!!

This is a bug?

With the Lizmap Webclient 3.5.1 version, is all ok!

Thank you.

Orlando

### Steps to reproduce the issue

![01_LWC_351](https://user-images.githubusercontent.com/15649016/160591298-e835a1cb-1a8f-42fc-8e32-4b770b0d5cc4.png)
01) draw a polygon (ver. 3.5.1)
![02_LWC_351](https://user-images.githubusercontent.com/15649016/160591301-918f5915-ba48-4b3c-9a5c-aba6d4cc1284.png)
02) click on the polygon for editing (ver. 3.5.1)
![03_LWC_351](https://user-images.githubusercontent.com/15649016/160591303-569ac288-cd29-4326-8182-fb5f990d6b73.png)
03) polygon edit (shows edit icons) (ver. 3.5.1)
![04_LWC_352](https://user-images.githubusercontent.com/15649016/160591304-0224aa5d-4925-421e-ab8d-bc53f89d4b84.png)
04) draw a polygon (ver. 3.5.2)
![05_LWC_352](https://user-images.githubusercontent.com/15649016/160591306-1c331266-0726-4d5a-84ba-4bb5c81df7ff.png)
05) click on the polygon for editing (ver. 3.5.2)
05) clicking on the edit icon does not show the respective icons for editing as in version 3.5.1 (image 03) (ver. 3.5.2)

### Versions

Publish and share your QGIS maps on the Web via Lizmap Web Client, by 3liz.com = Installed version | 3.7.6

### QGIS server version, only if the section above doesn't mention the QGIS Server version

3.16.3+28bionic server

### Operating system

Ubuntu 20.04

### Browsers

Firefox, Chrome

### Browsers version

Firefox 98.0.2 (64-bit); Chrome Version 99.0.4844.51 (Official Build) (64-bit)

**Comment by opbernardo** [comment]
Does anyone know if it's a bug?
What is the solution to this situation?
Thanks

**Comment by opbernardo** [comment]
Hi @Gustry,
I just update my issue.
Do you recommend to do a new installation for version 3.5.2, or just copy the folders "lib" and "lizmap" into the server?
Thank you for your help.
Orlando

**Comment by Gustry** [comment]
How have you done the update ?

Do you confirm that https://demo.snap.lizmap.com/lizmap_3_5/index.php/view/map/?repository=testsrepository&project=form_edition_simple_fields works as expected ?

**Comment by opbernardo** [comment]
Hi @Gustry,

Yes I confirm that works fine!


UPGRADE:

I extracted the version 3.5.2 zip files, then copied the bin and lizmap folders into my "/home/...../www/webgis/" folder and chose the overwrite option.

Then with filezilla I copied the same folders into the server "/home/qgis/www/webgis"

Is there another way to upgrade?

Without losing access to the various published projects?

Because if I do a fresh install of lizmap on"/home/...../www/testing" , I have to publish all the projects again, one by one

Thanks
